src/bo4e/enum/voraussetzungen.py

```diff
@@ -0,0 +1,76 @@
+    )
+    GEWERBE = "GEWERBE"  #: Gewerbenachweis
+    LASTPROFIL = "LASTPROFIL"  #: Kunde muss einem bestimmten Lastprofil zuzuordnen sein
+    ZAEHLERTYP_GROESSE = "ZAEHLERTYP_GROESSE"  #: bestimmter Zaehlertyp oder Zaehlergroeße
+    AUSSCHLUSS_GROSSVERBRAUCHER = "AUSSCHLUSS_GROSSVERBRAUCHER"  #: Ausschluss von Großverbrauchern wie Industriekunden oder produzierendes Gewerbe
+    NEUKUNDE = "NEUKUNDE"  #: Neukunden ohne bisherige Lieferanbindung
+    BESTIMMTE_VERTRAGSFORMALITAETEN = "BESTIMMTE_VERTRAGSFORMALITAETEN"  #: bestimmte Vertragsformalitaeten wie z.B. Anmeldeformular muss an bestimmte Adresse versandt werden
+    SELBSTABLESUNG = "SELBSTABLESUNG"  #: Selbstablesung durch den Kunden
+    ONLINEVORAUSSETZUNG = "ONLINEVORAUSSETZUNG"  #: Onlinevoraussetzung
+    MINDESTUMSATZ = (
+        "MINDESTUMSATZ"  #: Mindestumsatz in einer bestimmten Zeiteinheit wie z.B. Mindestjahresumsatz 2500 EURO
+    )
+    ZUSATZPRODUKT = (
+        "ZUSATZPRODUKT"  #: Zusatzprodukt zu bereits bestehendem Lieferverhaeltnis beim Versorger in dieser Energieart
+    )
+    NEUKUNDE_MIT_VORAUSSETZUNGEN = (
+        "NEUKUNDE_MIT_VORAUSSETZUNGEN"  #: geworbener Neukunde muss bestimmte Voraussetzungen erfüllen
+    )
+    DIREKTVERTRIEB = "DIREKTVERTRIEB"  #: Kunde wird durch Direktvertrieb gewonnen
+    ANSCHLUSSART = "ANSCHLUSSART"  #: Anlage mit bestimmter Anschlussart wie z.B. Festanschluss
+    ANSCHLUSSWERT = "ANSCHLUSSWERT"  #: bestimmte Anschlusswerte wie z. B. mindestens 30 kW
+    ALTER_KUNDENANLAGE = (
+        "ALTER_KUNDENANLAGE"  #: Alter einer Kundenanlage (z.B. Anlage wurde nach dem 01.01.2005 installiert)
+    )
+    ANLAGEBESCHAFFENHEIT = (
+        "ANLAGEBESCHAFFENHEIT"  #: bestimmte Anlagebeschaffenheit, wie bivalente Energieversorgung, Geräte o.ä.
+    )
+    BETRIEBSSTUNDENBEGRENZUNG = (
+        "BETRIEBSSTUNDENBEGRENZUNG"  #: Betriebsstundenbegrenzung z.B. max 1500h/a oder mindestens 1000h/a
+    )
+    FREIGABEZEITEN = "FREIGABEZEITEN"  #: vorgeschriebene Freigabezeiten
+    FAMILIENSTRUKTUR = (
+        "FAMILIENSTRUKTUR"  #: Familienstruktur wie z.B. bestimmte Anzahl Kinder oder Personen im Haushalt oder Eheleute
+    )
+    MITGLIEDSCHAFT = "MITGLIEDSCHAFT"  #: Mitgliedschaft in bestimmten Vereinen oder Verbaenden
+    STAATLICHE_FOERDERUNG = "STAATLICHE_FOERDERUNG"  #: staatliche Foerderung wie z.B. Sozialtarif oder Studentenausweis
+    BESONDERE_VERBRAUCHSSTELLE = (
+        "BESONDERE_VERBRAUCHSSTELLE"  #: besondere Verbrauchsstellen wie Kirchen, Vereinsgebaeude usw.
+    )
+    NIEDRIGENERGIE = "NIEDRIGENERGIE"  #: Niedrigenergieaustattung des Hauses
+    ORTSTEILE_LIEFERGEBIET = "ORTSTEILE_LIEFERGEBIET"  #: nur für bestimmte Ortsteile in diesem Liefergebiet
+    WAERMEBEDARF_ERDGAS = "WAERMEBEDARF_ERDGAS"  #: Waermebedarf wird nur oder überwiegend mit Erdgas gedeckt
+    MAX_ZAEHLER_LIEFERSTELLEN = "MAX_ZAEHLER_LIEFERSTELLEN"  #: beschraenkt auf max. Anzahl Zaehler oder Abnahmestellen
```

Comment:
Is `ä` a special character? ^^
```suggestion
    WAERMEBEDARF_ERDGAS = "WAERMEBEDARF_ERDGAS"  #: Wärmebedarf wird nur oder überwiegend mit Erdgas gedeckt
    MAX_ZAEHLER_LIEFERSTELLEN = "MAX_ZAEHLER_LIEFERSTELLEN"  #: beschränkt auf max. Anzahl Zähler oder Abnahmestellen
```